client/src/components/payment/upgrade-modal.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <v-dialog max-width="40%" :value="isVisible" persistent>
```

Comment:
Keep this in mind for future dialogs: you should pass a prop to them to "attach" to. In the parent you would pass a selector to something. This allows a dialog to be attached to the parent, so you can unit test it! (Otherwise it's attached to the DOM root which you can't see in the unit test.)

Also it would be great if components had a id or class, for testing purposes!

Reply:
I created a new tech debt ticket for this:
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5492